content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
 title: Quick Start
 toc: true
 type: docs
+description: >
+  This guide lets you quickly install Kmesh.
 
 ---
-## Cluster start mode
+This guide lets you quickly install Kmesh.
 
-### Kmesh container image prepare
+## Preparation
 
--   Kmesh achieves the ability to completely sink traffic management below the OS through kernel enhancements. When releasing images, the range of OS for which the image is applicable must be considered. To this end, we consider releasing three types of images:
+Kmesh needs to run on a Kubernetes cluster. Kubernetes 1.26, 1.27, 1.28 are currently supported. We recommend using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to quickly build a Kubernetes cluster. Of course, you can also use minikube and other ways to create Kubernetes clusters.
 
-    - Supported OS versions with kernel enhancement modifications
+Currently, Kmesh connects to the Istio control plane. Before starting Kmesh, install the Istio control plane software. For details, see [istio install guide](https://istio.io/latest/docs/setup/getting-started/#install).
 
-      The current [openEuler 23.03](https://repo.openeuler.org/openEuler-23.03/) OS natively supports the kernel enhancement features required by Kmesh. Kmesh release images can be directly installed and run on this OS. For a detailed list of supported OS versions with kernel enhancement modifications, please refer to [this link](https://github.com/kmesh-net/kmesh/blob/main/docs/kmesh_support.md).
-    
-    - For all OS versions:
+If you want to try out the Kmesh L7 feature as it requires waypoint, you need to install the [ambient mode istio](https://istio.io/latest/docs/ops/ambient/getting-started/).
```

Comment:
Kmesh L7 don't depend on ambient. It's available in non-ambient mode with some change of istiod configuration, it could be explained in the doc of L7